src/ui/pages/payment-entry-page.svelte

```diff
@@ -147,6 +147,16 @@
         taxCalculationStatus === "miss-match"),
   );
 
+  let expressCheckoutOptions = $derived(
+    subscriptionOption
+      ? StripeService.buildStripeExpressCheckoutOptionsForSubscription(
+          productDetails,
```

Comment:
We should pass the `priceBreakdown` instead of the `productDetails`. It contains the price in micros and also considers taxes